git-branchless-revset/src/builtins.rs

```diff
@@ -275,12 +316,24 @@ fn fn_stack(ctx: &mut Context, name: &str, args: &[Expr]) -> EvalResult {
 
 type MatcherFn = dyn Fn(&Repo, &Commit) -> Result<bool, PatternError> + Sync + Send;
 
-#[instrument(skip(f))]
+/// Make a pattern matcher that operates on all visible commits.
 fn make_pattern_matcher(
     ctx: &mut Context,
     name: &str,
     args: &[Expr],
     f: Box<MatcherFn>,
+) -> Result<CommitSet, EvalError> {
+    make_pattern_matcher_for_set(ctx, name, args, f, None)
+}
+
+#[instrument(skip(f))]
+/// Make a pattern matcher that operates only on the given set of commits.
```

Comment:
nit: reorder these two lines so that the doc-comment comes before any annotations.

Reply:
Oops, done!